packages/ui/src/CampaignRewards/CampaignRewardsComp.tsx

```diff
@@ -10,7 +10,7 @@ import Icon from 'ui/src/Icon'
 const RewardsCompSmall: React.FC<CampaignRewardsCompProps> = ({ rewardsPool, highContrast, mobile, banner }) => {
   const { platform, multiplier, platformImageSrc } = rewardsPool
 
-  const isPoints = rewardsPool.tags.includes('points')
+  const hasMultiplier = !!multiplier && multiplier !== ''
```

Comment:
`''` is falsey (i.e. `!!'' === false`)

```suggestion
  const hasMultiplier = !!multiplier
```